site/source/tutorials/1010_containers_and_injecting_state/index.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: false
+title: State management
+overview: In this tutorial, you will learn what a Containers and Injectors are and how to use them to manage external state and inject this into the parts of the widget tree.
```

Comment:
"and inject this into the parts of the widget tree" ... the reader will wonder if this refers to state, containers, or injectors